### PR TITLE
docs: Add new IMA document

### PR DESCRIPTION
**Pull request by cgwalters**
Now that the fixed code for `ima-sign` landed in
https://github.com/ostreedev/ostree-rs-ext/pull/283

### CODE REVIEW

**Comment by cgwalters** [comment]
:thinking: Hmm, I could imagine actually an ostree mode or format that actually *replaces* the ostree checksum with the (portable) IMA EVM checksum or so.  Would be quite nontrivial though.

File: docs/ima.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+## Further references
+
+- https://sourceforge.net/p/linux-ima/wiki/Home/
+- https://wiki.gentoo.org/wiki/Integrity_Measurement_Architecture
+- https://fedoraproject.org/wiki/Changes/Signed_RPM_Contents
+- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/enhancing-security-with-the-kernel-integrity-subsystem_managing-monitoring-and-updating-the-kernel
+
+
+###### Licensing for this document:
+`SPDX-License-Identifier: (CC-BY-SA-3.0 OR GFDL-1.3-or-later)`
```
Comment (jlebon) [inline docs/ima.md:111]: If you just want it in the markdown but unrendered, you can include it at the top in HTML quotes like this: `<!--- SPDX-License-Identifier: (CC-BY-SA-3.0 OR GFDL-1.3-or-later) --->` .

**Review by jlebon — APPROVED** [review]
Cool, this is very informative!

File: docs/ima.md (hunk 1 above)
Comment (cgwalters) [reply docs/ima.md:111]: This is in other documents, but I agree a comment is better.

**Review by jlebon — APPROVED** [review]

**Pull request merged**